docs/index.md

```diff
@@ -0,0 +1,137 @@
+# Overview
+
+Optimized einsum can significantly reduce the overall execution time of einsum-like
+expressions by optimizing the expression's contraction order and dispatching
+many operations to canonical BLAS, cuBLAS, or other specialized routines.
+Optimized einsum is agnostic to the backend and can handle NumPy, Dask,
+PyTorch, Tensorflow, CuPy, Sparse, Theano, JAX, and Autograd arrays as well as
+potentially any library which conforms to a standard API.
+
+## Features
+
+The algorithms found in this repository often power the ``einsum`` optimizations
+in many of the above projects. For example, the optimization of ``np.einsum``
+has been passed upstream and most of the same features that can be found in
+this repository can be enabled with ``numpy.einsum(..., optimize=True)``. However,
+this repository often has more up to date algorithms for complex contractions.
+Several advanced features are as follows:
+
+* Inspect [detailed information](paths/introduction.md) about the path chosen.
+* Perform contractions with [numerous backends](getting_started/backends.md), including on the GPU and with libraries such as [TensorFlow](https://www.tensorflow.org) and [PyTorch](https://pytorch.org).
+* Generate [reusable expressions](getting_started/reusing_paths.md), potentially with constant tensors, that can be compiled for greater performance.
+* Use an arbitrary number of indices to find contractions for [hundreds or even thousands of tensors](examples/large_expr_with_greedy.md).
+* Share [intermediate computations](./sharing_intermediates.md) among multiple contractions.
```

Comment:
This link also seems to be broken, needs to be ``getting_started/sharing_intermediates.md`` maybe?